tests/gen/README.md

```diff
@@ -0,0 +1,156 @@
+# Automatic Documents Generation 
```

Comment:
praise: I think this readme is very valuable and I thank you for taking the time to write it :)